docs/Coding_Conventions_and_Standards.md

```diff
@@ -23,6 +23,9 @@ Other
 * When adding a new class, disable copy/assignment/move until you have a proven need for these capabilities. If a need arises, enable copy/assignment/move selectively, and when doing so validate that the implementation of the class supports what is being enabled.
   * Use ORT_DISALLOW_COPY_ASSIGNMENT_AND_MOVE initially
   * See the other ORT_DISALLOW_* macros in https://github.com/microsoft/onnxruntime/blob/master/include/onnxruntime/core/common/common.h
+* Don't use else after return. see: [https://llvm.org/docs/CodingStandards.html#don-t-use-else-after-a-return](https://llvm.org/docs/CodingStandards.html#don-t-use-else-after-a-return)
+* Don't overuse std::shared\_ptr. Use std::shared\_ptr only if it's not clear when and where the object will be deallocated.
+* Avoid using the 'long' type, which could be either 32 bits or 64 bits.
```

Comment:
What about others like int or size_t? Is the recommendation to use the explicitly sized types like int32_t, or to avoid long in particular?

Reply:
Here I'm only referring 'long', in particular.

Reply:
Please feel free to add more if you think useful.